aws/cloudformation/scylla.yaml

```diff
@@ -56,6 +56,11 @@ Parameters:
     Type: 'AWS::EC2::KeyPair::KeyName'
     ConstraintDescription: must be the name of an existing EC2 KeyPair.
 
+  VpcCidrIp:
+    Description: Set CIDP IP range for VPC
+    Type: String
+    ConstraintDescription: must be vaild subnet
```

Comment:
s/vaild/valid/

Reply:
Is this sufficient, though? We have mentions of `172.31.x.x` IP addresses elsewhere in the CloudFormation template. Surely those need to be parametrised too for this to make sense?

/cc @fruch

Reply:
@penberg looks like we do need to parametrised those as well. but what if user will create a 5 node cluster?

Reply:
@fruch @penberg ping

Reply:
I'm not sure what the question here...

Reply:
in the template file we have those
```
          - - '{"scylla_yaml": {"seed_provider": [{"class_name": "org.apache.cassandra.locator.SimpleSeedProvider", "parameters": [{"seeds": "'
            - !Join
              - ','
              - - !If [Launch1, '172.31.0.11', !Ref "AWS::NoValue"]
                - !If [Launch2, '172.31.1.11', !Ref "AWS::NoValue"]
                - !If [Launch3, '172.31.2.11', !Ref "AWS::NoValue"]
 ```
So if a user is using different subnet, we should probably also modify those values  ,right ?